index.html

```diff
@@ -198,6 +198,30 @@ <h3>Expression Format</h3>
           [=preference=] for the Global Privacy Control value.
         </p>
       </section>
+      <section>
+        <h3>Preference Caching</h3>
+        <p>
+          The [=preference=] MUST be cached on every top-level navigation to ensure consistency in communication of
+          the person's request that their data "not be sold or shared." This means that if the [=preference=] changes
+          during or after a top-level navigation, it WILL NOT be reflected until the next navigation.
+        </p>
+        <p>
+          A [=top-level browsing context=] has a <dfn><code>gpcAtNavigation</code></dfn> boolean.
+          It is initially <code>false</code>.
+        </p>
+        <p>
+          When [=process a navigate fetch=] is run with [=top-level browsing context=] as <code>browsingContext</code>,
+          [=top-level browsing context=]'s <dfn><code>gpcAtNavigation</code></dfn> is updated to
+          <code>true</code> if the [=preference=] is enabled, and <code>false</code> if the the [=preference=] is disabled
```

Comment:
"the the" should be "the"

Draft editors can make sure we use consistent language throughout. I think we want to say true "if the person's preference is set" and false "if the person's preference is disabled or has not been set."